blog-article.html

```diff
@@ -185,269 +187,266 @@ <h1 class="intro__title">
+                    </li>
+                </ul>
+                <ul class="pointer-list">
+                    <li>
+                        List name 1
+                    </li>
+                    <li>
+                        List name 2
+                    </li>
+                    <li>
+                        List name 3
+                    </li>
+                    <li>
+                        List name 4
+                    </li>
+                </ul>
+                <div class="blockquote">
+                    <blockquote>
+                        "The first huge metaverse arena to arrange live sports, hang out and even make bets and wagers"
+                    </blockquote>
+                    <cite class="cite">
+                        <span>8,5 / 10</span>
+                        Markout Corporation
+                    </cite>
+                </div>
+                <div class="post__img">
+                    <figure>
+                        <img src="img/face-dragon.jpg" width="448" height="390" alt="dragon-face">
+                        <figcaption>Photo caption</figcaption>
+                    </figure>
+                    <figure>
+                        <img src="img/big-dragon.jpg" width="448" height="390" alt="dragon-big">
+                        <figcaption>Photo caption</figcaption>
+                    </figure>
+                </div>
+                <h3 class="post__title">NFTs are digital assets that represent real-world objects</h3>
+                <p>
+                    Setting up your avatar and personalizing your nickname on Binance is quick and easy. Users can go to binance.com under Dashboard > Settings > My Profile > Avatar and Nickname. For those of you who own NFT assets, you can select the NFT you want to use from your collection as your avatar to personalize your profile. For users who do not own any NFT avatars, check out the NFT avatar collections page to get started.
+                </p>
+                <p>
+                    Alternatively, if you’re on nft.binance.com, go to User Center > Settings > Basic > Set Avatar and Nickname, and you’ll be redirected to the main site. Binance NFT Creators have the option of setting their Avatars on nft.binance.com. Once you set your new unique nickname, it will be standardized and used across the Binance ecosystem.
+                </p>
+                <h3 class="post__title">
+                    NFT Avatar Generators
+                </h3>
+                <p>
+                    You can find a range of NFT marketplaces where you can buy existing NFTs or list your own NFTs for sale.
+                </p>
+                <footer class="article-footer">
```

Comment:
Actually, footer on the index and this page looks the same. Probably we can try to use the same classname